src/Microsoft.AspNetCore.Server.IISIntegration/Server/IISHttpContext.ReadWrite.cs

```diff
@@ -12,6 +12,8 @@ namespace Microsoft.AspNetCore.Server.IISIntegration
 {
     internal partial class IISHttpContext
     {
+        private const int StackChunkLimit = 128;
```

Comment:
Seems to small, even for 256k stack 1k should be fine.

Reply:
I see, it's a stuct, how big is it?

Reply:
Units?

Reply:
I think we can increase it. The struct size is only 16 bytes: https://github.com/aspnet/HttpSysServer/blob/231cbffe1a5cc0abce11e2d0685683ed74fc7ad5/shared/Microsoft.AspNetCore.HttpSys.Sources/NativeInterop/HttpApiTypes.cs#L78

Reply:
128 is good if it's a segment count.